### PR TITLE
added user guide, ssa, release notes

### LINKED ISSUE #1227: Convert Docs To Markdown
**Is your feature request related to a problem? Please describe.**
See https://github.com/pokt-network/organization/blob/main/milestones/1-08-01-2021/6.0/milestone.md
For details

### PR DESCRIPTION
**Pull request by andrewnguyen22**
closes #1227

### CODE REVIEW

File: doc/user guide.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+MaxTxsBytes": Max size of ALL Txs in bytes
+* **"CacheSize"**: Max memory cache size of mempool in transactions.
+* **"MaxTxBytes"**: Max size of Tx in bytes
+
+#### FastSync (only if main/fastsync_mode=true)
+Fast Sync version to use:
+* **"Version"**: "v1"
+- "v0" - the legacy fast sync implementation
+- "v1" (default) - refactor of v0 version for better testability
+- "v2" - complete redesign of v0, optimized for testability & readability
```
Comment (edjroz) [inline doc/user guide.md:242]: Should we mark v2? as far as I know in our current tendermint fork v2 is non operational.

File: doc/user guide.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+**Force Unstake**
+
+If a Validator falls below the minimum stake (due to slashing) it will be forcibly removed by the protocol and all Staked Tokens burned. This feature of the protcol highlights the importance of staking 'well above' the minimum stake.
+
+**Economic Incentives**
+
+For providing infrastructure access to applications, Validators are rewarded proportional to the work they provide. Pocket Core attempts to send a *Claim* and subsequent *Proof* transaction automatically after the `proof_waiting_period` elapses. If both transactions are successful, Tokens are minted to the address of the Validator.
+
+## Resources
+[Pocket Core CLI Spec](cli-interface-spec.md)
```
Comment (edjroz) [inline doc/user guide.md]: This is now contained and split into a cli-interface-spec directory rather than a single file see #1232

**Review by edjroz — COMMENTED** [review]
Fantastic doc overview, I would split up the user guide, as its a big file makes it harder for users to quickly look up what they want.

I think the user guide has 1 issue which is no explanation for how fee's affect transactions (minimum, incentive to use higher fee, etc...) other than in the fund your account section.

File: doc/user guide.md (hunk 1 above)
Comment (andrewnguyen22) [reply doc/user guide.md:242]: interesting, but actually, the default is still v1

File: doc/user guide.md (hunk 2 above)
Comment (andrewnguyen22) [reply doc/user guide.md]: Well I guess it depends which PR is merged first haha JK, I'll fix it

**Comment by andrewnguyen22** [comment]
> I think the user guide has 1 issue which is no explanation for how fee's affect transactions (minimum, incentive to use higher fee, etc...) other than in the fund your account section.

Good point

**Pull request merged**